core/google/cloud/client.py

```diff
@@ -22,6 +23,13 @@
 from google.cloud.credentials import get_credentials
 
 
+_GOOGLE_AUTH_CREDENTIALS_HELP = (
+    'This library only supports credentials from google-auth-library-python. '
```

Comment:
Nit: Aren't newlines good for this text? Wouldn't it be better to use a triple-quoted / multiline string?